sass/_PrintableUri.scss

```diff
@@ -1,21 +1,33 @@
 @import "Variables";
 
-.CoveoPrintableUri{
+.CoveoPrintableUri {
+  justify-content:flex-start;
   margin: 2px 0 5px 0;
-
-
-  a {
-	font-size: $font-size-smallest;
-	@include clickable();
-	padding: 0 2px 0 2px;
-	display:inline-block;
-  }
-
+  font-size: $font-size-smallest;
+  @include clickable();
+  padding: 0 2px 0 2px;
+  display: flex;
```

Comment:
Using bourbon mixin on every flex rule, but I noticed that the ones for the flex-box have a deprecation warning. Leave them as is or use bourbon for that too ?